src/entities/DFGVariable.jl

```diff
@@ -93,7 +95,7 @@ mutable struct DFGVariable <: AbstractDFGVariable
     label::Symbol
     timestamp::DateTime
     tags::Vector{Symbol}
-    estimateDict::Dict{Symbol, Dict{Symbol, <: AbstractVariableEstimate}}
+    estimateDict::Dict{Symbol, <: AbstractVariableEstimate}
```

Comment:
have we figured out if we going with `<: AbstractParametricPointEst` yet?

see: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/147#issuecomment-545639185

Reply:
you are using it above, so might just be a mistake or am i missing something?

Reply:
Just missed it thanks. will fix